components/server/src/user/user-service.ts

```diff
@@ -74,7 +74,7 @@ export class UserService {
         }
         if (newUser.additionalData) {
             // When a user is created, it does not have `additionalData.profile` set, so it's ok to rewrite it here.
-            newUser.additionalData.profile = { acceptedPrivacyPolicyDate: new Date().toISOString() };
+            // todo:revert newUser.additionalData.profile = { acceptedPrivacyPolicyDate: new Date().toISOString() };
```

Comment:
what should I do with this todo? Revert immediately?

Reply:
Also wouldn't new users get the notification now as well, because seeing the terms during signup is not recorded?